edb/server/protocol/auth_ext/data.py

```diff
@@ -80,3 +80,69 @@ def __repr__(self) -> str:
             f"iss={self.iss!r} "
             f"email={self.email!r})"
         )
+
+
+@dataclasses.dataclass
+class OpenIDConfig:
```

Comment:
Is this dataclass too fat? We only care about a few of these properties, maybe I should just list those and ignore the others?

Reply:
Yeah, we can always add them back when needed, right?